zerver/lib/markdown/fenced_code.py

```diff
@@ -237,19 +248,35 @@ def done(self) -> None:
         self.processor.pop()
 
 
+def parse_hl_lines(hl_lines: Optional[str] = None) -> List[int]:
+    """Support our syntax for emphasizing certain lines of code.
+    hl_lines should be like '1 2' to emphasize lines 1 and 2 of a code block.
+    Returns a list of ints, the line numbers to emphasize.
+    """
+    if not hl_lines:
+        return []
+
+    try:
+        return list(map(int, hl_lines.split()))
+    except ValueError:  # nocoverage
```

Comment:
Can we add a test covering this?

Reply:
Or if we think that's impossible, maybe we should make it a `raise AssertionError` instead?